cli/docker/docker.go

```diff
@@ -102,6 +102,19 @@ func ExecCommandIntoContainer(ctx context.Context, containerName string, user st
 
 	output = strings.ReplaceAll(output, "\n", "")
 
+	patterns := []string{
```

Comment:
my binary is a little rust, What is it for?

Reply:
ok, I saw it https://github.com/elastic/e2e-testing/pull/299/files#r489219873

Reply:
I guess the docker client is returning something weird when converting the bytes. Maybe I'm doing something weird stuff with the buffer and the reader. I'd need to debug it.